sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/trees/TreeNode.scala

```diff
@@ -604,6 +604,7 @@ abstract class TreeNode[BaseType <: TreeNode[BaseType]] extends Product {
     }.toList
   }
 
+  // TODO: Fix toJSON so that we can more safely handle Map and Seq with loop.
```

Comment:
I think this comment deserves to have an example. Also, it will be good to just create a jira with your example.